R/cache.R

```diff
@@ -77,7 +77,7 @@ Cache <- R6::R6Class(
       if (missing(value)) {
         ls(private$cachedObjects)
       } else {
-        stop(messages$errorPropertyReadOnly("keys"), call. = FALSE)
+        stop(ospsuite.utils::messages$errorPropertyReadOnly("keys"), call. = FALSE)
```

Comment:
We had decided to not use the namespace operator for the utilities package to reduce verbosity and to slightly increase readability. This is reverting changes made in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/717.